### ISSUE TITLE
Add `list-props` prop and expand the item slot props for `c-select`

### ISSUE THREAD

**Issue by kelseymcmahon** [IntelliTect/Coalesce#655]
I am having accessibility issues with the `v-combobox` that require access to some props to fix the errors. The specific errors I need to fix are the missing label on the `v-combobox` list ([rules](https://dequeuniversity.com/rules/axe/4.11/aria-input-field-name?application=AxeChrome)) and removing the focus from the check boxes inside of a list ([rules](https://dequeuniversity.com/rules/axe/4.11/nested-interactive?application=AxeChrome)). The following is an example of a c-select that I had to convert to a `v-combobox` so I could fix the accessibility errors they were causing:

```
<c-select
    v-model="myModel"
    for="ModelName"
    :params="{ dataSource: myDataSource}"
    color="primary"
    multiple
    clearable
    closable-chips
    hide-details
    chips
  />
```

```
 <v-combobox
    v-model="myModel"
    v-model:search="myList.$search"
    :items="myList.$items"
    :loading="myList.$load.isLoading"
    item-title="name"
    color="primary"
    label="Cases to Invite"
    multiple
    clearable
    closable-chips
    hide-details
    chips
    return-object
    :list-props="
      { ariaLabel: 'List of items', tabindex: 0 } as any
    "
  >
    <template #item="{ props }">
      <v-list-item v-bind="props ">
        <template #prepend="{ isSelected, select }">
          <v-list-item-action
            start
            aria-hidden="true"
            inert
          >
            <v-checkbox-btn
              :model-value="isSelected"
              @update:model-value="select"
            />
          </v-list-item-action>
        </template>
      </v-list-item>
    </template>
  </v-combobox>
``` 

If we could add the `:list-props` prop to the c-select or c-input and also add the props from the item slot ([link](https://vuetifyjs.com/en/api/v-combobox/#slots-item)) then I could easily fix these issues where needed.
